rats-tls/dist/rpm/rats_tls.spec.in

```diff
@@ -37,6 +37,30 @@ ExclusiveArch: x86_64
 %description
 rats-tls is a protocol to establish secure and trusted channel by integrating enclave attestation with transport layer security.
 
+%package host
+Summary: host mode package for rats-tls
+
+%description host
+rats-tls-host is the "host" build mode package of rats-tls.
+
+%package occlum
+Summary: occlum mode package for rats-tls
+
+%description occlum
+rats-tls-occlum is the "occlum" build mode package of rats-tls.
+
+%package sgx
+Summary: sgx mode package for rats-tls
+
+%description sgx
+rats-tls-sgx is the "sgx" build mode package of rats-tls.
+
+%package tdx
```

Comment:
According to our last discussion, rats-tls provides three build modes: host sgx and occlum.

Reply:
I remember we can drop "tdx" mode when sgx sdk combines sgx and tdx in header and lib files.
I already file an issue to record such requirement. https://github.com/inclavare-containers/inclavare-containers/issues/1544
Currently, if you drop tdx, tdx package will not generated and it will be a mismatch with README. So that's why i keep it. 
If you insist to remove it now, i suggest you also need to drop it in README.